applications/GeoMechanicsApplication/custom_workflows/time_loop_executor.hpp

```diff
@@ -106,8 +112,16 @@ public :
         return mCancelDelegate && mCancelDelegate();
     }
 
+    void UpdateProgress(double Time)
```

Comment:
This function can be const (see also [SonarCloud](https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&branch=geo%2Fsettlement-progress-delegate&id=KratosGeomechanics&open=AYv4hfTGfW0A5I6bHWgc), which is a nice tool to find these kinds of 'code-smells')